client/entry/index.js

```diff
@@ -0,0 +1,3 @@
+// This is intentionally empty entry point.
```

Comment:
What's the purpose of this empty entry point?

Reply:
The main purpose is so that the loginedInMiddleware works as expected for the jetpack cloud.